package.json

```diff
@@ -19,7 +19,7 @@
     "url": "https://github.com/gtramontina/ghooks/issues"
   },
   "scripts": {
-    "lint": "eslint bin/ lib/ test/",
+    "lint": "eslint bin/ lib/* test/",
```

Comment:
`hook.template.raw`did not get linted before (`npm test` would have failed, because that file did not comply to the eslint rules yet).